docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -928,6 +928,10 @@ yb-voyager export data from source --export-dir <EXPORT_DIR> \
         --export-type snapshot-and-changes
 ```
 
+{{< note title="PostgreSQL and parallel jobs" >}}
+For PostgreSQL, make sure that no other processes are running on the source database that can lead to issues with more than one parallel job, where Voyager will not take locks to dump the data.
+{{< /note >}}
+
```

Comment:
Add this note to offline migration page under `Export data` heading.

modify this 

> where Voyager will not take locks to dump the data.

to 

> where Voyager will not be able to take locks to dump the data.